src/components/Dashboard.js

```diff
@@ -1,15 +1,12 @@
-import React, { Component } from 'react';
+import React from 'react';
 import SideNavigation from './SideNavigation';
 /**
  * Component for rendering the main page
  */
-class Dashboard extends Component {
-  render() {
-    return (
-      <div>
```

Comment:
All all these changes still about lint?

Reply:
This was a class component and eslint highlighted that it should be a function component.
So I had to rectify that.